dtbootstrap-anima-keyinfra.xml

```diff
@@ -713,12 +713,32 @@ INSERT_TEXT_FROM_FILE time-sequence-diagram.txt END
           </list></t>
       </section>
       <section title="Cloud Registrar">
-          <t>The Pledge MAY contact a well known URI of a cloud Registrar if a
+        <t>
+          There are transitional situations where devices may be
+          deployed into legacy networks that use proprietary
+          bootstrapping mechanisms based upon the base EST (<xref
+          target="RFC7030"/>).  The same device may also be deployed
+          into an ANIMA environment.  This may be due to incremental
+          replacement of a legacy situation with ANIMA.
+        </t>
+        <t>
+          There are additionally some greenfield situations involving an
+          entirely new installation where a device may have some kind of
+          management uplink that it can use (such as via 3G network for
+          instance).   In such a future situation, the device might use
+          this management interface to learn that it should
+          configure itself by to-be-determined mechanism (such as an Intent)
+          to become the local registrar.
```

Comment:
A simpler case could be, 

“There exist operationally open network wherein device gains unauthenticated access to the internet at large. In these use cases the management domain for the device needs to be discovered within the larger internet. These are less likely within the anima scope but may be more important in the future."

Reply:
Good.  put it in then?